### PR TITLE
fix: restore docs.scss to resolve docs links overlapping

### PR DESCRIPTION
**Pull request by mmaoyu**
Restore docs.scss to resolve version less than v1.9 docs links overlapping. The docs.scss was deleted in commit [ae7e444](https://github.com/rook/rook.github.io/commit/ae7e444b02efd48534de3e86ae0a5ab50fd6f022)

### CODE REVIEW

**Comment by mmaoyu** [comment]
Local test result: 
image
image

File: css/docs.scss [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+        color: #ffffff;
+      }
+    }
+
+    &.old {
+      border-color: #2196f3;
+      color: #2196f3;
+    }
+  }
+}
```
Comment (parth-gr) [inline css/docs.scss:432]: leave a extra line

File: css/docs.scss [hunk 2]
```diff
@@ -0,0 +1,432 @@
+---
+---
```
Comment (parth-gr) [inline css/docs.scss:2]: can you show the output of broken page that was in the issue

**Review by parth-gr — CHANGES_REQUESTED** [review]

File: css/docs.scss (hunk 2 above)
Comment (mmaoyu) [reply css/docs.scss:2]: Yeah, I've updated  the issue comment

File: css/docs.scss (hunk 1 above)
Comment (mmaoyu) [reply css/docs.scss]: done

**Comment by travisn** [comment]
> @mmaoyu Does this simply restore the css deleted in #156? Or are there any changes?

The diff from previous css looks empty, i'll go ahead and merge, thanks!

**Pull request merged**